include/dp_session.h

```diff
@@ -65,6 +65,60 @@ enum dp_session_state {
 	SESSION_STATE_CLOSED,
 } __attribute__ ((__packed__));
 
+/**
+ * Session attribute.
+ */
+enum dp_session_attr {
```

Comment:
Is the need to request specific attrs due to the plugin directly outputting the returned JSON?
The plugin should retrieve the necessary attrs from the JSON and format them into a suitable table.
Therefore it should not matter if additional attrs are returned.
The dataplane could return all of the attrs, and the plugin only output the relevant values.

Reply:
The plugin registers a session_watch to hook when session state or stats changed. This was called from dataplane forwarding path and I try to keep plugin not affect forwarding performance as much as possible.

I see that returning all attrs from dataplane make easier for adding new attributes but I afraid it could affect performance. That why I added a query as parameter.